doc/man7/openssl-core.h.pod

```diff
@@ -69,20 +69,175 @@ It's normally passed in arrays, where the array is terminated with an
 element where all fields are zero (for non-pointers) or C<NULL> (for
 pointers).
 
-These arrays can be used both to set parameters for some object, and
-to request parameters.
+These arrays can be used both to pass parameter data to some function,
+but can also be used to request data from some function.
+For a request, this description talks about a requesting function that
+provides the C<OSSL_PARAM> array, and a responding function that fills
+in the buffers that the C<OSSL_PARAM> elements point at.
 
```

Comment:
It looks to me like you kept and older version of this file. All you seem to add here is also in OSSL_PARAM.pod, isn't it?

Reply:
yes, because I started from Pauli's original branch which didn't sync.  Updated to master.